tests/LSP7DigitalAsset/extensions/LSP7CompatibleERC20.behaviour.ts

```diff
@@ -519,44 +543,41 @@ export const shouldInitializeLikeLSP7CompatibleERC20 = (
     });
 
     it("should have set expected entries with ERC725Y.setData", async () => {
-      await expect(context.initializeTransaction).toHaveEmittedWith(
+      await expect(context.initializeTransaction).to.emit(
         context.lsp7CompatibleERC20,
-        "DataChanged",
-        [SupportedStandards.LSP4DigitalAsset.key]
-      );
+        "DataChanged"
+      ).withArgs(SupportedStandards.LSP4DigitalAsset.key);
       expect(
         await context.lsp7CompatibleERC20["getData(bytes32)"](
           SupportedStandards.LSP4DigitalAsset.key
         )
-      ).toEqual(SupportedStandards.LSP4DigitalAsset.value);
+      ).to.be.equal(SupportedStandards.LSP4DigitalAsset.value);
 
       const nameKey =
         "0xdeba1e292f8ba88238e10ab3c7f88bd4be4fac56cad5194b6ecceaf653468af1";
```

Comment:
I suggest that we use the values from `constants.js` and avoid hex literals in tests for data keys, as these are important (also to just defined them in one place, if they ever need to be changed).

add at the top:

```ts
import { ERC725YKeys } from "../../constants.js"
```

and change by

```suggestion
       ERC725YKeys.LSP4["LSP4TokenName"]
```

or 

```suggestion
        ERC725YKeys.LSP4.LSP4TokenName
```

> I prefer the 1st one to differentiate the LSP standard and the data key, but it is up to you, both syntax are valid :)